src/main/java/frc/robot/subsystems/shooter/commands/Shoot.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.shooter.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+import java.util.function.DoubleSupplier;
+
+public class Shoot extends CommandBase {
+    private final Shooter shooter;
+    private final DoubleSupplier velocitySetpoint;
+
+    /**
+     * Constructor.
+     *
+     * @param shooter          is the shooter subsystem.
+     * @param velocitySetpoint is the setpoint for the shooter wheel. [rps]
+     */
+    public Shoot(Shooter shooter, DoubleSupplier velocitySetpoint) {
+        this.shooter = shooter;
+        this.velocitySetpoint = velocitySetpoint;
+        addRequirements(shooter);
+    }
+
+    @Override
+    public void execute() {
+        shooter.setVelocity(Shooter.getSetpointVelocity(velocitySetpoint.getAsDouble()));
```

Comment:
WHAT?

Reply:
what?

Reply:
please rename `velocitySetpoint`, it seems that you convert velocity to another velocity (not distance to velocity)

Reply:
Also you don't need to convert distance to velocity, look at Outtake for how you can do that without it...